packages/terra-application-utility/src/utility/_UtilityButton.jsx

```diff
@@ -135,7 +140,7 @@ class ApplicationHeaderUtility extends React.Component {
   }
```

Comment:
on line 96, create a new line and add `menuRole`.

Reply:
I think that may lead to a lint error as we'd destructure menuRole and I think the linter would complain about menuRole being defined but never used, not 100% sure though.

Reply:
ye, your suggestion of deleting the prop makes more sense 👍